modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/ckeditor5/ClassicEditor.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {CKEditor} from '@ckeditor/ckeditor5-react';
+import {
+	Bold,
+	ClassicEditor as BaseClassicEditor,
+	EditorConfig,
+	Essentials,
+	GeneralHtmlSupport,
+	Italic,
+	Paragraph,
+	Style,
+	StyleConfig,
+	Underline,
+} from 'ckeditor5';
+import React from 'react';
+
+import Timestamp from './plugins/Timestamp';
```

Comment:
My original intent was to implement custom plugin in the sample module, pass it in as a prop, and just add it is in the `plugins` or `extraPlugins` array here. The idea was that product teams could own plugins, with code in their modules. What surprised me is that custom plugins need to be initiated only in one module. There is a mechanism that does not allow multiple initialization of plugins, you end up with [duplicated modules error](https://ckeditor.com/docs/ckeditor5/latest/support/error-codes.html#error-ckeditor-duplicated-modules). Plugins code can still be in a different module, but is would need to be exported and imported here.

But the question here is not just a technical one. Do we even want to promote plugins owned by product teams?  It might be a good idea for FI to simply own them all. If that is the case, there is no reason not to have them in the same module, like in this PR.